pandas/_libs/lib.pyx

```diff
@@ -134,8 +154,9 @@ def is_scalar(val: object) -> bint:
             or util.is_period_object(val)
             or is_decimal(val)
             or is_interval(val)
-            or util.is_offset_object(val))
-
+            or util.is_offset_object(val)
+            or isinstance(val, Number)
```

Comment:
you can use a compound ``isinstance(val, (Number, Fraction))``

Reply:
oh! I didn't know that !
Thanks @jreback